upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -229,41 +218,12 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
-state_name="UPDATE_BSS_CLOUD_INIT_RECORDS"
+state_name="UPDATE_CLOUD_INIT_RECORDS"
 state_recorded=$(is_state_recorded "${state_name}" $(hostname))
 if [[ $state_recorded == "0" && $(hostname) == "ncn-m001" ]]; then
     echo "${state_name} ..."
 
-    # get bss cloud-init data with host_records
```

Comment:
According to @SeanWallace , the rest of these need to stay here (lines 237-261).   Only the call to ncn-upgrade-cloud-init.sh should be removed.   Correct me if I misunderstood you, Sean.